components/create_post/create_post.jsx

```diff
@@ -574,18 +593,18 @@ export default class CreatePost extends React.Component {
         post.create_at = time;
         post.parent_id = this.state.parentId;
```

Comment:
I only have one concern. Here we are populating the post with some extra data (parentId, time, userId...) probably that things make a lot of sense in the hook, I think we should move it to doSubmit too.